### ISSUE TITLE
Complete start guide

### ISSUE THREAD

**Issue by astrowalker** [sjefvanleeuwen/blazor-adminlte#53]
I would like to thank you for your work, however even the best projects do not shine without proper documentation like "starter guide".

So could you provide correct and complete steps assuming user/dev created Blazor solution? What to do next?

Just in case somebody would think I am lazy, I am not, here are my steps, but I doubt they are 100% solid.

* download zip file from https://adminlte.io/ 
* unpack it
* copy dist and plugins directory to your wwwroot directory (in solution)
* install Blazorized.AdmintLte and Blazorized.AdmintLte.Content
* copy entire content of index.html page as described on the main page
* same thing for MainLayout
* fix all the links removing AdminLte prefix
* in Program.Main of hosting project add
```
  builder.Services.AddAdminLte();
```
* donwload interops.js from https://raw.githubusercontent.com/sjefvanleeuwen/blazor-adminlte/master/src/Blazor.AdminLte/wwwroot/js/interop.js 
* download site.css from https://raw.githubusercontent.com/sjefvanleeuwen/blazor-adminlte/master/src/Blazor.AdminLte.Site.Shared/wwwroot/css/site.css 
* download manifest json from https://raw.githubusercontent.com/sjefvanleeuwen/blazor-adminlte/master/src/Blazor.AdminLte.Wasm/wwwroot/manifest.json  
* after downloading fix all the keys as it is invalid json file -- quotation marks are missing
* download icon-512.png  from https://raw.githubusercontent.com/sjefvanleeuwen/blazor-adminlte/master/src/Blazor.AdminLte.Wasm/wwwroot/icon-512.png 
* donwload service-worker.js https://raw.githubusercontent.com/sjefvanleeuwen/blazor-adminlte/master/src/Blazor.AdminLte.Wasm/wwwroot/service-worker.js  -- please note it is placed directly in the wwwroot

At this point solution works without displaying any errors, but I am afraid that considering the amount of manual work I missed some automatic step and thus I had to this.

So in short, start guide would be great addition to this project :-).

**Comment by jornandojunior** [comment]
this tutorial video is complete.
https://www.youtube.com/watch?v=ILTrxpehdnQ

**Comment by twojnarowski** [comment]
It is not so easy for me. I get errors.
`RZ9985	Multiple components use the tag 'PageTitle'. Components: Microsoft.AspNetCore.Components.Web.PageTitle, Blazor.AdminLte.PageTitle`
and:
`CS0103 name "routeData" does not exists in current context.`

So I had also to change:
`<PageTitle>Index</PageTitle>`
to:
`<Blazor.AdminLte.PageTitle Title="Index" />`
In all .razor pages.

But when I try to copy also some elements, I got an error that "BreadcrumbsContainer" elements does not exists.
